--- context.py.orig	2016-11-28 10:25:25.889083596 -0500
+++ context.py	2016-11-28 10:26:26.496404041 -0500
@@ -304,6 +304,7 @@
             exists in the returned set.  If the project_name parameter is non-None and there is NOT
             a match in the returned set raise an Exception stating that fact.
         """
+        self._currentProject = project_name
         result = self.agent.get('Project', fetch="Name", workspace=self._currentWorkspace, project=None)
 
         if not result or result.resultCount == 0:
