--- context.py.orig	2016-10-25 10:50:34.190730693 -0400
+++ context.py	2016-10-25 10:51:13.599602240 -0400
@@ -154,12 +154,14 @@
 ##
         target_host = proxy_host or server
 
+        """
         if self.server_ping:
             reachable, problem = Pinger.ping(target_host)
             if not reachable:
                 if not problem:
                     problem = "host: '%s' non-existent or unreachable"  % target_host
                 raise RallyRESTAPIError(problem)
+        """
 
         user_response = self._getUserInfo()
         subscription = self._loadSubscription()
@@ -293,6 +295,7 @@
             exists in the returned set.  If the project_name parameter is non-None and there is NOT
             a match in the returned set raise an Exception stating that fact.
         """
+        self._currentProject = project_name
         result = self.agent.get('Project', fetch="Name", workspace=self._currentWorkspace)
 
         if not result or result.resultCount == 0:
